dataactbroker/handlers/fileHandler.py

```diff
@@ -1328,7 +1337,9 @@ def serialize_submission(submission):
         "reporting_start_date": str(submission.reporting_start_date),
         "reporting_end_date": str(submission.reporting_end_date),
         "user": {"user_id": submission.user_id,
-                 "name": submission_user_name}
+                 "name": submission_user_name},
+        "certifying_user": certifying_user,
+        'publish_status': submission.publish_status.name,
```

Comment:
This is part of the serialization because it's intended to be displayed in the submission tables, correct?